maqs-playwright/src/test/java/com/cognizantsoftvision/maqs/playwright/PageDriverFactoryUnitTest.java

```diff
@@ -55,4 +59,11 @@ public void browserWithNoContext() {
     PageDriver pageDriver = PageDriverFactory.getPageDriverFromBrowser(browser);
     Assert.assertFalse(pageDriver.getAsyncPage().isClosed());
   }
+
+  @Test(groups = TestCategories.PLAYWRIGHT)
+  public void defaultOptionsUseProxy() {
+    Config.addTestSettingValues(Collections.singletonMap("UseProxy", "Yes"),
```

Comment:
This is a very very very bad idea.
You override a pretty low level setting and just leave it set - I am guessing this is why so many tests started failing after you added this test.